### PR TITLE
updated docs and added test to examples of kdeplot.

### PR DESCRIPTION
**Pull request by pyeguy**
as @mmagnuski  pointed out this should have been in #1186

### CODE REVIEW

File: seaborn/distributions.py [hunk 1]
```diff
@@ -451,8 +451,8 @@ def kdeplot(data, data2=None, shade=False, vertical=False, kernel="gau",
 
     Parameters
     ----------
-    data : 1d array-like
-        Input data.
+    data : 1d array-like or 2d array-like
```
Comment (mwaskom) [inline seaborn/distributions.py]: This form of input is being deprecated.

File: ' [hunk 2]
```diff
@@ -0,0 +1,12 @@
+white space fix in example section.
```
Comment (mwaskom) [inline ']: What is this file?

**Comment by mwaskom** [comment]
I don't think it's necessary, thanks.

**Comment by pyeguy** [comment]
No worries, felt like overkill

**Pull request closed without merging**